sotodlib/tod_ops/pca_box.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
+from tqdm import tqdm
+import matplotlib.pyplot as plt
+
+import sotodlib
+from sotodlib import hwp
+from sotodlib.tod_ops.detrend import detrend_tod
+from sotodlib.tod_ops import flags, pca
+from sotodlib.core.flagman import has_any_cuts, has_all_cut, count_cuts
+from preprocess_pca_box import load_amans, load_metas, jump_amans, lpf_amans
+
+import logging
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Do we need this submodule anymore? It seems like this stuff has been moved into `preprocess.preprocess_plot.ploc_pcabounds` and `tod_ops.pca.calc_pcabounds` if so delete this.

Reply:
Nope, I left the module in for potential debugging use.  All lingering comments are likely mine, as I didn't clean up from debugging prior to re-review.